samples/dotnet/DistributedTracing/KafkaTriggerManyWithHeaders.cs

```diff
@@ -0,0 +1,43 @@
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Kafka;
+using Microsoft.Azure.WebJobs.Extensions.Storage;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+
+namespace DistributedTracing
+{
+    public class KafkaTriggerManyWithHeaders
+    {
+        [FunctionName("KafkaTriggerManyWithHeaders")]
+        public static void Run(
+            [KafkaTrigger("BrokerList",
+                          "topic",
+                          Username = "$ConnectionString",
+                          Password = "%EventHubConnectionString%",
+                          Protocol = BrokerProtocol.SaslSsl,
+                          AuthenticationMode = BrokerAuthenticationMode.Plain,
+                          ConsumerGroup = "$Default")] KafkaEventData<string>[] events, ILogger log)
+        {
+            foreach (KafkaEventData<string> eventData in events)
+            {
+                log.LogInformation($"C# Kafka trigger function processed a message: {eventData.Value}");
+                log.LogInformation($"Headers: ");
+                var headers = eventData.Headers;
+                foreach (var header in headers)
+                {
+                    log.LogInformation($"Key = {header.Key} Value = {System.Text.Encoding.UTF8.GetString(header.Value)}");
+                }
+            }
```

Comment:
Why are we printing headers?

Reply:
If the traceparent is present in the headers, then the trace id for the current activity would be the same as that in the traceparent. This is for the customer to see that the trace id  in current activity and that present in headers is same.

Reply:
Instead of printing all the headers, Will it make sense to print just the traceparent from the headers?

Reply:
Done.